dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -2518,4 +2518,82 @@ open class Jpql : JpqlDsl {
     fun <T : Any> deleteFrom(entity: Entityable<T>): DeleteQueryWhereStep<T> {
         return DeleteQueryDsl(entity.toEntity())
     }
+
+    /**
+     * Creates an expression that represents the position of the first occurrence of a substring in a string.
+     * If the substring is not found, returns 0. position start with 1.
+     */
+    @SinceJdsl("3.0.0")
+    fun locate(
```

Comment:
Since it would be nice to have a similar position between expressions, I would like to declare the expression method above the function method.

https://github.com/line/kotlin-jdsl/pull/485/files#diff-9b44577a7f634570b16728fdb4fd998398fc979e19a5b316eff4b80e3eea0049R1049

Reply:
리뷰 반영했습니다. 적절한 위치 잡기가 애매했는데. 감사합니다.